benchmarks/src/main/scala/scalaz/zio/QueueParallelBenchmark.scala

```diff
@@ -38,19 +54,14 @@ class QueueParallelBenchmark {
 
   @Benchmark
   def fs2Queue(): Int = {
```

Comment:
I was surprised to see that backpressured benchmark showed much higher throughput than parallel, even for fs2:
```
Backpressure
[info] Benchmark                             Mode  Cnt     Score       Error  Units
[info] QueueBackPressureBenchmark.fs2Queue  thrpt    3    51.871 ±     2.813  ops/s
[info] QueueBackPressureBenchmark.zioQueue  thrpt    3  3658.171 ± 13240.533  ops/s

Parallel
[info] Benchmark                         Mode  Cnt    Score     Error  Units
[info] QueueParallelBenchmark.fs2Queue  thrpt    3    1.354 ±   0.101  ops/s
[info] QueueParallelBenchmark.zioQueue  thrpt    3  452.826 ± 106.976  ops/s
```
Do you have an ideas why it could be so?

Reply:
Yeah there is something wrong. I had the following results before:
```
Benchmark                                 Mode    Cnt    Score    Error  Units

QueueSequentialBenchmark.fs2Queue         thrpt    3    1.223 ±   0.140  ops/s
QueueSequentialBenchmark.zioQueue[OLD]    thrpt    3    1.648 ±   1.382  ops/s
QueueSequentialBenchmark.zioQueue[NEW]    thrpt    3  252.721 ±  15.430  ops/s

QueueParallelBenchmark.fs2Queue           thrpt    3    1.022 ±   0.077  ops/s
QueueParallelBenchmark.zioQueue[OLD]      thrpt    3  149.937 ±   8.778  ops/s
QueueParallelBenchmark.zioQueue[NEW]      thrpt    3  685.956 ± 129.736  ops/s

QueueBackPressureBenchmark.fs2Queue       thrpt    3   30.665 ±  68.346  ops/s
QueueBackPressureBenchmark.zioQueue[OLD]  thrpt    3   66.530 ±  15.322  ops/s
QueueBackPressureBenchmark.zioQueue[NEW]  thrpt    3   80.792 ±   9.859  ops/s
```
I'll try to find what caused that difference.

Reply:
Probably `threadPool.allowCoreThreadTimeOut(true)` - might be doing something funny and mess with the benchmark.

Reply:
Yeah, it seems it's the fact I override `threadPool` that causes the problem. I tried to override it with the exact same content as `newDefaultThreadPool` in RTS and there's the same problem. If I remove that override it works well.

Reply:
BTW I see in your message above in results for fs2: BackPressure.fs2 = 30 > Parallel.fs2 = 1. Do you have an idea why backpressured fs2 scores lower?

Reply:
No I haven't looked at their implementation. Overall I imagine it's not designed for atomic operations like we do here and is probably more efficient with the stream methods.